wazo_auth_client/types.py

```diff
@@ -35,4 +35,9 @@ class TokenDict(TypedDict):
     remote_addr: str
 
 
+class SSODict(TypedDict):
```

Comment:
```suggestion
class SSOResponseDict(TypedDict):
```

would be more explicit as to what this is describing